src/openapi/streaming/streaming.ts

```diff
@@ -400,6 +402,13 @@ class Streaming extends MicroEmitter<EmittedEvents> {
 
         switch (this.connectionState) {
             case this.CONNECTION_STATE_DISCONNECTED:
+                log.info(
+                    LOG_AREA,
+                    'Connection disconnected',
+                    { contextMessageCount: this.contextMessageCount },
+                    { persist: true },

```

Comment:
Need to figure out how often this happens and decide if it (and the below call) should be persisted, to avoid spamming ourselves

Reply:
we did persist it until recently. Can we just check if we log it inside saxotrader too.
So I think its ok.

Reply:
In saxotrader it has some spam mitigation for disconnections since it checks if a timeout is already in progress before logging  - do you think we'll need something similar here or we'll be alright logging on every disconnection?